source/packages/libraries/clients/assetlibrary-client/src/di/types.ts

```diff
@@ -10,7 +10,7 @@
  *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions    *
  *  and limitations under the License.                                                                                *
  *********************************************************************************************************************/
-export const ASSTLIBRARY_CLIENT_TYPES = {
+export const ASSETLIBRARY_CLIENT_TYPES = {
```

Comment:
If we need to avoid the breaking change while correcting the typo, you could add

`export const ASSTLIBRARY_CLIENT_TYPES = ASSETLIBRARY_CLIENT_TYPES` 
until there is another breaking change

However, that might be just as confusing as the original typo. 😄

Reply:
@TonySherman that works. If we can add a comment as "old type will be deprecated at somepoint" should cover it. Also your PR need change logs.

Reply:
Added support for legacy users with comment. Included change logs as well.